src/legacy/ui/public/agg_types/metrics/lib/parent_pipeline_agg_controller.js

```diff
@@ -1,76 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import _ from 'lodash';
-import { i18n } from '@kbn/i18n';
-
-const parentPipelineAggController = function ($scope) {
-  $scope.$watch('responseValueAggs', updateOrderAgg);
-  $scope.$watch('agg.params.metricAgg', updateOrderAgg);
-
-  $scope.$on('$destroy', function () {
-    const lastBucket = _.findLast($scope.state.aggs, agg => agg.type && agg.type.type === 'buckets');
```

Comment:
This logic was moved to `MetricAggParamEditor` control  https://github.com/elastic/kibana/pull/39502/files#diff-d09c7d350e3017c52e2e151fba97ef27R108